src/App.tsx

```diff
@@ -1,10 +1,49 @@
-import React from 'react';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import React, { useEffect, useRef } from 'react';
+import { Alert, AppState } from 'react-native';
 import { Provider } from 'react-redux';
 
 import AppContainer from './AppContainer';
 import { store } from './store';
 
 const App = () => {
```

Comment:
What do you think about moving logic to separate file?
same way like AppGate works.

Reply:
this could be good idea